src/mumble/GlobalShortcut.cpp

```diff
@@ -548,6 +549,24 @@ QString ShortcutDelegate::displayText(const QVariant &item, const QLocale &loc)
 	return QStyledItemDelegate::displayText(item, loc);
 }
 
+bool ShortcutDelegate::helpEvent(QHelpEvent *event, QAbstractItemView *, const QStyleOptionViewItem &,
+								 const QModelIndex &index) {
+	if (event->type() != QEvent::ToolTip || !index.isValid()) {
+		return false;
+	}
+
+	QString text;
+	QTextStream textStream(&text);
+	textStream << tr("Shortcut button combination:") << '\n';
+
+	const auto shortcuts = index.data(Qt::DisplayRole).toList();
+	textStream << GlobalShortcutEngine::buttonText(shortcuts, QChar('\n'));
```

Comment:
```suggestion
	textStream << GlobalShortcutEngine::buttonText(shortcuts, QString::fromLatin1("\n-"));
```
Does this produce a nice list-syntax in the tooltip? Something like
```
Shortcut button combination:
- <first btn>
- <second btn>
```

I think this might be nice to have. If it doesn't work this way, I don't think that this is important, so feel free to merge without these changes.

Reply:
I was thinking about it as well. We would have to use HTML to achieve a list similar to the GitHub Markdown one.

Reply:
Doesn't the solution with the simple dashes work? Imo we don't have to go full-fancy and have proper bullets in front of the items. Just some sort of enumeration indicator might be useful :thinking:

Reply:
Looks much better than without.